Script/lib/maneuver-lib.ks

```diff
@@ -15,7 +15,7 @@ LOCAL FUNCTION get_normal {
 // accepts individual ship values instead of the ship
 // so that burn time can be calculated based on future state
 //
-// PARAMETER dv: deltav in m/s for the node
+// PARAMETER dv0: deltav in m/s for the node
```

Comment:
Apparently `dv` is reserved for something or is somehow in use somewhere else, and was becoming `0`.